charts/m4d/values.yaml

```diff
@@ -1,24 +1,135 @@
-# Default values for m4d.
+# Default values for Mesh for Data.
 # This is a YAML-formatted file.
 # Declare variables to be passed into your templates.
 
 global:
-  opa: true
-  opaConnector: true
-
-image:
-   mover: ghcr.io/the-mesh-for-data/mover:latest
-   manager: ghcr.io/the-mesh-for-data/manager:latest
-   secretProvider: ghcr.io/the-mesh-for-data/secret-provider:latest
-   opaConnector: ghcr.io/the-mesh-for-data/opa-connector:latest
-   opaConnectorPullPolicy: IfNotPresent
-
-# extraArgs to be passed to manager container
-extraArgs:
-  - --enable-leader-election
-  - --enable-all-controllers
-
-# extraEnvs to be set for manager container
-extraEnvs: []
-# - name: env_name
-#   value: env_value
+  # Default hub for Mesh for Data images.
+  hub: ghcr.io/the-mesh-for-data
+  # Default tag for Mesh for Data images.
+  # If no value is set, the chart's appVersion will be used.
+  tag: ""
+  # Default image pull policy for Mesh for Data images
+  # Default behavior: latest images will be Always else IfNotPresent.
+  # ref: https://kubernetes.io/docs/concepts/containers/images/#updating-images
+  imagePullPolicy: ""
+  # Reference to one or more secrets to be used when pulling images
+  # ref: https://kubernetes.io/docs/tasks/configure-pod-container/pull-image-private-registry/
+  imagePullSecrets: []
+
+manager:
+  # Image name or a hub/image[:tag]
+  image: "manager"
+  # Overrides global.imagePullPolicy
+  imagePullPolicy: ""
+
+  # extraArgs to be passed to manager container
+  extraArgs:
+    - --enable-leader-election
+    - --enable-all-controllers
+
+  # extraEnvs to be set for manager container
+  extraEnvs: []
+  # - name: env_name
+  #   value: env_value
+
+mover:
+  # Image name or a hub/image[:tag]
+  image: "mover"
+  # Overrides global.imagePullPolicy
+  imagePullPolicy: ""
+
+secretProvider:
+  # Image name or a hub/image[:tag]
+  image: "secret-provider"
+  # Overrides global.imagePullPolicy
+  imagePullPolicy: ""
+
+opaConnector:
+  # Set to false to skip deploying OPA connector
+  enabled: true
+  # Image name or a hub/image[:tag]
+  image: "opa-connector"
+  # Overrides global.imagePullPolicy
+  imagePullPolicy: ""
+
+opaService:
+  # Set to false to skip deploying OPA service
+  enabled: true
+  # Image name or a hub/image[:tag]
+  image: "openpolicyagent/opa:latest"
+  # Overrides global.imagePullPolicy
+  imagePullPolicy: ""
+
+katalogConnector:
```

Comment:
is the term katalogConnector is correct ? should it be katalogService instead? i belive it is not just the connector but also the CRD is derived from these flags.

Reply:
I would like to keep xxxConnector for consistency. You comment implies that we should also add `katalog` for the CRDs/RBAC, which I considered but decided that it will just complicate UX because currently it has no use outside m4d when the connector is enabled.